release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -298,6 +298,89 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
 [id="ocp-4-14-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-14-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.9.2
+* node-exporter to 1.6.1
+* prom-label-proxy to 0.7.0
+* Prometheus to 2.46.0
+* prometheus-operator to 0.67.1
+
+[id="ocp-4-14-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `NodeSystemSaturation` alert to monitor resource saturation on a node.
+** Added the `NodeSystemdServiceFailed` alert to monitor the systemd service on a node.
+** Added the `NodeMemoryMajorPagesFaults` alert to monitor major page faults on a node.
+** Added the `PrometheusSDRefreshFailure` alert to monitor failed Prometheus service discoveries.
+
+* *Changed*
+** Modified the `KubeAggregatedAPIDown` alert and the `KubeAggregatedAPIErrors` alert to evaluate only metrics from the `apiserver` job.
+** Modified the `KubeCPUOvercommit` alert to evaluate only metrics from the `kube-state-metrics` job.
+** Modified the `NodeHighNumberConntrackEntriesUsed`, `NodeNetworkReceiveErrs` and `NodeNetworkTransmitErrs` alerts to evaluate only metrics from the `node-exporter` job.
+
```

Comment:
MultipleContainersOOMKilled is removed from 4.14, see:
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.13/assets/cluster-monitoring-operator/prometheus-rule.yaml
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.14/assets/cluster-monitoring-operator/prometheus-rule.yaml

Reply:
@jan--f Same question as above re dropped alerts

Reply:
This one is probably worth mentioning. We removed this alert since it was a stop gap until the node team introduced the relevant metric for the container runtime. They have and decided that adding a OOM alert was not benefital. See https://issues.redhat.com/browse/OCPNODE-600. Instead they alert on available memory running out.

Reply:
Added `MultipleContainersOOMKilled` to the list under "Removed".